src/components/Create/components/pages/ReviewDeploy/components/RewardsReview/RewardsReview.tsx

```diff
@@ -50,5 +58,24 @@ export const RewardsReview = () => {
     [dispatch],
   )
 
-  return <RewardsList value={rewards} onChange={setRewards} />
+  const useDataSourceForRedeem = useMemo(() => {
```

Comment:
nit - I think we should avoid `use` prefix for anyting that isn't a hook. Pretty low chance of someone confusing this, but just for good practice

Reply:
Nah great call, addressed 👍